pkgs/tools/audio/yabridge/default.nix

```diff
@@ -79,6 +79,13 @@ in stdenv.mkDerivation rec {
   )'';
 
   patches = [
+    # Fix for wine 6.8+ (remove patch in next release):
+    (fetchpatch {
```

Comment:
`fetchpatch` should be added back as an argument after `fetchFromGitHub`.

Reply:
Oops, fixed.